webapp/src/components/shortcut.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled, {css} from 'styled-components';
+
+import {isMac} from 'src/shortcuts';
+
+type Props = {
+    shortcut: string,
+};
+
+const macKeyToCharMap: {[key: string]: string} = {
+    meta: '⌘',
+    alt: '⌥',
+};
+
+export default function Shortcut(props: Props) {
+    const keys = props.shortcut.split('+');
+
+    const renderKeys = () => {
+        return keys.map((key) => {
+            let ch = '';
+            if (isMac()) {
+                ch = macKeyToCharMap[key];
+            }
+            if (!ch) {
+                ch = key;
+            }
```

Comment:
minor nit: would it be cleaner to do this?
```suggestion
        return keys.map((key) => {
            let ch = key;
            if (isMac()) {
                ch = macKeyToCharMap[key];
            }
```

Reply:
It would be cleaner but semantic would be different as `macKeyToCharMap` is expected to potentially return undefined if the char is not one of those mapped (only those two atm) in which case we want to fallback. So we would have to at least check that in addition.